### ISSUE TITLE
Allow import of tables from Kibana

### ISSUE THREAD

**Issue by Alex-At-Home** [Alex-At-Home/elasticsearch-sheets#55]
This is do-able but a bit tricky

First given the ID eg at the start of the URL path you can get the visualization object, eg 

`GET .kibana/visualization:<id>`

you can get the index pattern from there via `GET .kibana/index-pattern:<id2>` having pulled `<id2>` out of the search object

The first object (you could also pull all that stuff out of the URL but that would require keeping a Kib URL <-> JSON parser up-to-date) gives a stable-looking list of aggregations

The second object gives the index pattern and in theory the query (together with the URL) .. a better solution would be to also get the user to paste the contents of "Inspect > Requests > Request" in, which has the query in it

So in summary:
* index pattern: `URL -> vis ID -> JSON -> grab index ID -> JSON -> string`
* query: `inspect UI`
* aggs: `URL -> vis ID -> JSON -> grab list of aggs/col names` AND`inspect UI -> grab list of aggregations, correlate via agg ids with col names`
